Program.cs

```diff
@@ -1,22 +1,26 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Price_Calculator_kata
+﻿namespace Price_Calculator_kata
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            TaxServices MyTaxService= new TaxServices();
-            PriceCalculator MyPriceCalculator = new PriceCalculator(MyTaxService);
+
+            var products = new ProductRepository();
+
+            var MyDiscountService = new DiscountService();
+
+            TaxServices MyTaxService = new TaxServices();
+            PriceCalculator MyPriceCalculator = new PriceCalculator(MyTaxService, MyDiscountService);
```

Comment:
This is so so so great !
now this step will help us do unit testing very very easily in the future ! injective the services will make replacing/mocking them very easy in case of testing them
this is called LSP: https://reflectoring.io/lsp-explained/, to go further with this, I added comments above to add interface to these service (Tax + Discount)